.github/workflows/image_build_push.yml

```diff
@@ -27,34 +27,21 @@ jobs:
     - name: Set docker image tag from .env file
       run: |
         set -a; source .env; set +a
+        echo "Restoring latest server image tag from .env"
         echo "DOCKER_TAG_FROM_PUSH=${SERVER_IMAGE_TAG}" >> $GITHUB_ENV
 
+    - name: Set docker image tag from .env.repoTags file
+      run: |
+        set -a; source .env.repoTags; set +a
+        echo "Restoring latest frontend image tag from .env.repoTags"
+        echo "FRONTEND_IMAGE_TAG=${FRONTEND_IMAGE_TAG}" >> $GITHUB_ENV
```

Comment:
I am not going to insist on this, but we should revisit this as part of 
https://github.com/e-mission/e-mission-docs/issues/1082

" Consider switching from $GITHUB_ENV to step outputs. I think that the step outputs are the recommended approach to pass values from one step to another, but we should verify"